spaces/yieldfarming-insure/skin.scss

```
@@ -0,0 +1,10 @@
+.name {
```

This must be the key of your space
```suggestion
.yieldfarming-insure {
```

---

If you haven't changed the default style you don't need to add this file

---

thanks, file removed